examples/CMakeLists.txt

```diff
@@ -67,4 +68,4 @@ target_link_libraries(test_host matx::matx)
 
 if(MATX_EN_PYBIND11)
     add_subdirectory(python_integration_sample)
-endif()
\ No newline at end of file
+endif()
```

Comment:
Seems like a mistake?

Reply:
Code formatter added a newline at end of file, let me try to minimize changes....

Reply:
I split this (and a few other minor) changes out into their own PR, just so that this PR is purely focused on the new sparse functionality, without any distractions....